Google.PowerShell/BigQuery/GcbqDataset.cs

```diff
@@ -20,8 +21,8 @@ namespace Google.PowerShell.BigQuery
     /// READER dataset role. The -All flag will include hidden datasets in the search results. The -Filter 
     /// flag allows you to filter results by label. The syntax to filter is "labels.name[:value]". Multiple filters 
     /// can be ANDed together by connecting with a space. See the link below for more information.
-    /// If no Project is specified, the default project will be used. This cmdlet returns a DatasetList if 
-    /// no DatasetId was specified, and a Dataset otherwise.
+    /// If no Project is specified, the default project will be used. This cmdlet returns any number of 
+    /// DatasetList.DatasetData objects if no DatasetId was specified, and a Dataset otherwise.
```

Comment:
Alright, take a look now.  Had a minor issue with pester last night so I didn't get the changes up until now.